src/retriers.js

```diff
@@ -0,0 +1,14 @@
+export let retriers = {
+  is5xx: (response) => {
+    if(response && response.status &&
+      response.status >= 500 &&
+      response.status < 600) {
+        return true;
+    }
+    return false;
+  },
+
+  isNetworkError: (response) => {
+    return response instanceof Error;
```

Comment:
Oops! I thought we were calling it on the error thrown by json-fetch. It is ok then 👍